File: shojiwm/src/input.ts
import { COMPOSITOR } from "shoji_wm";

export function configureInput(): void {
  COMPOSITOR.input.configure((input) => {
    input.global = {
      keyboard: {
        layout: "us,ru",
        variant: ",",
        options: "grp:alt_shift_toggle",
        repeatRate: 25,
        repeatDelay: 300,
      },
      pointer: {
        accelProfile: "flat",
      },
    };
  });
}

File: hypr/config/input.lua
hl.config({
    input = {
        kb_layout = "us,ru",
        kb_variant = "",
        kb_options = "grp:alt_shift_toggle",
        repeat_rate = 25,
        repeat_delay = 300,
        accel_profile = "flat",
    },
})

hl.gesture({ fingers = 4, direction = "horizontal", action = "workspace" })
hl.gesture({ fingers = 3, direction = "down", action = "close" })
hl.gesture({ fingers = 3, direction = "up", action = "fullscreen" })
hl.gesture({ fingers = 3, direction = "left", action = "float" })
